src/webpage/Cargo.lock

```diff
@@ -550,6 +550,12 @@ version = "0.21.7"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "9d297deb1925b89f2ccc13d7635fa0714f12c87adce1c75356b39ca9b7178567"
 
+[[package]]
```

Comment:
Please merge the cargo lock commits

Reply:
There are two cargo lock files, do you want me to merge the two commits?